common/lib/dependabot/pull_request_creator.rb

```diff
@@ -216,19 +220,32 @@ def codecommit_creator
     end
 
     def message
-      @message ||=
-        MessageBuilder.new(
-          source: source,
-          dependencies: dependencies,
-          files: files,
-          credentials: credentials,
-          commit_message_options: commit_message_options,
-          pr_message_header: pr_message_header,
-          pr_message_footer: pr_message_footer,
-          vulnerabilities_fixed: vulnerabilities_fixed,
-          github_redirection_service: github_redirection_service,
-          dependency_group: dependency_group
-        )
+      return @message unless @message.nil?
+
+      case source.provider
+      when "github"
+        @pr_message_max_length = Github::MAX_PR_DESCRIPTION_LENGTH if @pr_message_max_length.nil?
```

Comment:
Let's call all of these constants `PR_DESCRIPTION_MAX_LENGTH` as I think "pr description" is the parent in the taxonomy... it also matches better with `@pr_message_max_length`

Reply:
I had used the naming convention that was already in place, but based on your suggestion I have updated the names to your proposed convention.